tasks/strings/strings.js

```diff
@@ -4,23 +4,38 @@
  * @param {string} str
  */
 function reverseString(str) {
+  if (typeof (str) == 'number') return 'This is not a string!';
+  let arr = str.split('');
+  let arrReverse = arr.reverse();
+  let strReverse = arrReverse.join('');
+  return strReverse;
```

Comment:
one-line:
return str.split('').reverse().join('');

Reply:
fixed,
but it is very difficult to perceive yet these long constructions)